src/panels/lovelace/entity-rows/hui-input-text-entity-row.ts

```diff
@@ -54,7 +53,6 @@ class HuiInputTextEntityRow extends LitElement implements LovelaceRow {
       <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
         <paper-input
           no-label-float
-          .disabled=${UNAVAILABLE_STATES.includes(stateObj.state)}
```

Comment:
We should keep the check for unavailable, unknown should be removed I agree.

Reply:
Input entities can't really be unavailable, though, since there's no real connection to anything. If it exists at all, it has a value and is available. Otherwise it'll display the yellow error bar.

Reply:
I have them in my dev instance... not sure how I got them... are restored ones

Reply:
But are they unavailable, or is that their state?

Reply:
I.E. can you change their state from the more-info card or a service call.

Reply:
Ideally, there should probably be two discrete... things. `state` and `value`.

I can see more things benefiting from that, but it's a huge change.

Reply:
Probably if you remove them from your yaml config, the entity reg is not cleaned up.

Reply:
Ok. You've convinced me.